src/fontra/views/editor/scene-controller.js

```diff
@@ -1148,6 +1168,49 @@ export class SceneController {
     });
   }
 
+  async closeContour() {
```

Comment:
Somehow closing doesn't work when the open path starts with a curve:
<img width="236" alt="image" src="https://github.com/googlefonts/fontra/assets/4246121/84a0b519-5f52-4e61-87bb-ececc7b5f2d6">
Or maybe it's the smooth flag that prevents it to work.

Reply:
I fixed it, with the help of the below comment.